theme/src/primer-nav.yml

```diff
@@ -20,6 +20,8 @@
       url: https://primer.style/css
     - title: React
       url: https://primer.style/react
+    - title: React Brand
+      url: https://primer.style/brand/
```

Comment:
Nitpicking, but we don't have a trailing slash on the other urls

Reply:
As I hit the merge button but I can make another PR to remove it

Reply:
I just realised that we're missing a changeset too. Can you add one in please in the follow-up?

`npx changeset`

then follow commands.

I _think_ version should be `patch` IMO, but feel free to suggest otherwise. cc. @colebemis